pywbem_mock/_wbemconnection_mock.py

```diff
@@ -19,13 +19,10 @@
 #
 
 """
-Mock WBEMConnection to allow pywbem users to test the pywbem client without
-requiring a running WBEM server. This package mocks the WBEMConnection
-methods that communicate with WBEMServers and mantain a fake server
-repository that is the source and destination for information to
-be used by the methods mocked.
+Mock support for the WBEMConnection class to allow pywbem users to test the
+pywbem client without requiring a running WBEM server.
 
-For the module-level documentation, see mocksupport.rst.
+For documentation, see mocksupport.rst.
```

Comment:
I did not notice this before but we are pointing to a rst file instead of to the external documentation,  Shouldn't this point to the build doc, moc_support section

Reply:
This module docstring is not generated into the external documentation, so I guess it does not matter.